index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Week1</title>
+    <link rel="stylesheet" href="styles/style.css">
+</head>
+<body>
+    <header>
+        <nav>
+            <ul>
+                <li><a href=""> <img src="images/swiggylogo.png" alt = "swiggylogo" class = "swiggy-logo"> </a></li>
+                <li> <a href="" class="setloc">Set location</a> </li>
+                <li class="chooseloc"> <a href="">Click to Choose Location</a> </li>
+            </ul>
+            <ul>
+                <li><a href="">Help</a></li>
+                <li><a href="">Sign In</a></li>
+                <li><a href="">Cart</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <section class = "home">
+        <nav>
+            <ul>
+                <li><a href="">Home</a> <small>/</small></li>
```

Comment:
HTML elements have a semantic meaning and should not be used purely for presentational purposes.
You can use a span here
`<span class="small-text"></span>`